Sponsor carousel #45

Mikiyas-hs opened the pull request:
## What does this change?

Resolves issue #45 

For this user story, I created the part of the application where users can view the sponsorship partners. I built an automatic carousel using only CSS, so you can easily see all the sponsors. In this video, you can see how it looks in action.


https://github.com/user-attachments/assets/013c0036-0565-4727-a5f2-45480bdf7434



## How Has This Been Tested?
<!-- Link to test results in the Wiki-->

- [ ] [User test]()
- [ ] [Accessibility test]()
- [ ] [Performance test]()
- [ ] [Responsive Design test]()
- [ ] [Device test]()
- [ ] [Browser test]()

## Images

<!-- Usually only applicable to UI changes, what did it look like before and what will it look like after? -->


## How to review

If you open this branch and start the localhost, you’ll be able to see my components as soon as you click the link.
I would like your feedback on my code:

Am I following proper coding conventions?
Is it clear and understandable?
I would like your feedback on the design:

Does anything look off?
Could anything be improved?
Did I forget anything?

vsheo committed:
Create CONTRIBUTING.md

vsheo committed:
Update CONTRIBUTING.md

team canvas, afspraken en code conventies

vsheo committed:
Update CONTRIBUTING.md

vsheo committed:
Merge branch 'main' into development

vsheo committed:
sveltekit build setup #19

vsheo committed:
folders & files for routes #19

vsheo committed:
netlify adaptor #19

vsheo committed:
Update README.md instalation instuctions

vsheo committed:
Update CONTRIBUTING.md git flow, naamgeving & coventional commits

vsheo committed:
Update CONTRIBUTING.md issues assignen

vsheo committed:
Merge pull request #20 from fdnd-agency/dev-branch-setup-19

Dev branch setup 19

vsheo committed:
figma colors in project #35

vsheo committed:
build: font files & responsive fonts #35

vsheo committed:
fix: 2 classes #35

vsheo committed:
fix: removed comments #35

vsheo committed:
Update CONTRIBUTING.md

development branch changed to ddev branch

vsheo committed:
Update CONTRIBUTING.md

verduidelijking dat we in dev branch werken

vsheo committed:
docs: readme titles

vsheo committed:
docs: project description & design choices

vsheo committed:
fix: colors more accurate #35

Pink-Mallow committed:
Update CONTRIBUTING.md [RETROSPECT 1] (Sprint 14)

We have done the retrospect with a Retromat:
1. Set the stage - Round of admiration (#76)
2. Gather Data - The good the bad and the ugly 
3. Generate insights - Wish granted
4. Decide what to do - Pitch 
5. Close the retrospective - You and me 

We have discussed all these five points by looking back at Sprint 14. 
After that we made team agreements on everything we all can agree on. 
At last we also discussed on what we want to work on in cooperation in the next Sprint.

Pink-Mallow committed:
Update CONTRIBUTING.md

Small documentation changes on the retrospective part

Pink-Mallow committed:
Update CONTRIBUTING.md

vsheo committed:
DoR & DoD added into contributing

vsheo committed:
dod & dor refinements

vsheo committed:
Merge pull request #36 from fdnd-agency/style-guide-colors--35

figma colors in project #35

this pull request has been approved, I will now merge this into the dev branch

vsheo committed:
fix: removed static font sizes #43

vsheo committed:
fix: fix names in custom properties #43

vsheo committed:
Merge branch 'dev' into styleguide-fonts-35

vsheo committed:
Merge pull request #43 from fdnd-agency/styleguide-fonts-35

Styleguide fonts 35

the pull request has been approved and the merge conflicts have been resvolved

vsheo committed:
fix: class names for colormix #43

vsheo committed:
fix: accidently left out code in the last merge #43

ive added te left out code

vsheo committed:
Merge pull request #60 from fdnd-agency/styleguide-fonts-35

Styleguide fonts 35 fix

Mikiyas-hs committed:
Sponsor carousel HTML (#45)

Mikiyas-hs committed:
Sponsor carousel (CSS) (#45)

Mikiyas-hs committed:
Sponsor carousel (Carousel-animatie) (#45)

Mikiyas-hs committed:
Sponsor images (#45)

Mikiyas-hs committed:
images

Mikiyas-hs committed:
Data images (#45) (#79)

Handmatige ingevoerde images veranderd voor images die worden opgehaald vanuit directur

vsheo reviewed (changes requested):
Feedback from my review of the code and files:
- The assets for the logos can be deleted, as they now come from Directus

Feedback on requested review points:
- your code is understandable
- in your commit mesages you are not following the [conventional commit types](https://docs.fdnd.nl/conventies.html#conventional-commits) yet
- the animation looks great, but when i have prefers reduced motion the ul stops at a wierd spot  
<img width="2850" height="745" alt="image" src="https://github.com/user-attachments/assets/a110e4a9-9dde-4ff1-b0d8-37ea5d729526" />  

- is anything off? yes, sometimes there is a gap in the ul (between li elements)
- For improvements: You could stop the animation on hover.
  - Also, consider making the ul element scrollable when the user hovers and attempts to scroll.

Mikiyas-hs committed:
chore: remove commented-out sponsor carousel code (#45) (#79)

Cleaned up old commented code that was replaced by the new carousel structure

Mikiyas-hs committed:
refactor: rename variable q to sponsor for better readability (#45) (#79)

Improved code readability by replacing unclear variable name q with a more descriptive one

Mikiyas-hs committed:
Refactor: replace custom :root colors with custom properties (#45) (#79)

Removed custom :root colors and replaced them with custom properties from our stylesheet

(--color-neutral, etc)

Mikiyas-hs committed:
refactor: scope layout and styling to the sponsor section instead of body (#45) (#79)

Removed global body styles (margin, overflow, background etc) and applied them only to the sponsors section to prevent conflicts with other components in +page.svelte and future +layout.svelte

Mikiyas-hs committed:
Refactor: clean up sponsor section (#45)

Mikiyas-hs committed:
chore: got rid of the width (#45) (#79)

the section already was full width by itself

Mikiyas-hs committed:
Merge branch 'main' into sponsor-carousel
